size_report fails in encodings\cp1252.py with cmd.exe (works with PowerShell)

I'm using nRF Connect SDK v2.0.2.
Machine is Windows 10.
Before calling `west` commands, I've ran `C:\ncs\v2.0.2\zephyr\zephyr-env.cmd` with console to set needed environment variables.
Then I change directory to the root folder of my app and run `west build -b <some_board>`.
After succesfull build, I've ran `west build -t rom_report` which, after 15 seconds or so, results in FATAL ERROR shown below:
```sh
-- west build: running target rom_report
[0/1] cmd.exe /C "cd /D C:\Users\Korisnik\Desktop\ivan\projekti\art_nr1amp\build && C:\ncs\toolchains\v2.0.2\opt\bin\python.exe C:/ncs/v2.0.2/zephyr/scripts/footprint/size_report -k C:/project/build/zephyr/zephyr.elf -z C:/ncs/v2.0.2/zephyr -o C:/project/build --workspace=C:/ncs/v2.0.2 -d 99 rom"
Path                                                                                             Size    %
==============================================================================================================
Root                                                                                           212424 100.00%
Traceback (most recent call last):
  File "C:/ncs/v2.0.2/zephyr/scripts/footprint/size_report", line 803, in <module>
    main()
  File "C:/ncs/v2.0.2/zephyr/scripts/footprint/size_report", line 792, in main
    print_any_tree(root, symsize, args.depth)
  File "C:/ncs/v2.0.2/zephyr/scripts/footprint/size_report", line 697, in print_any_tree
    print(f"{row.pre}{cc}{row.node.name} {s} {cr}{Fore.BLUE}{percent:6.2f}%{Fore.RESET}")
  File "C:\ncs\toolchains\v2.0.2\opt\bin\lib\site-packages\colorama\ansitowin32.py", line 41, in write
    self.__convertor.write(text)
  File "C:\ncs\toolchains\v2.0.2\opt\bin\lib\site-packages\colorama\ansitowin32.py", line 162, in write
    self.write_and_convert(text)
  File "C:\ncs\toolchains\v2.0.2\opt\bin\lib\site-packages\colorama\ansitowin32.py", line 187, in write_and_convert
    self.write_plain_text(text, cursor, start)
  File "C:\ncs\toolchains\v2.0.2\opt\bin\lib\site-packages\colorama\ansitowin32.py", line 195, in write_plain_text
    self.wrapped.write(text[start:end])
  File "encodings\cp1252.py", line 19, in encode
UnicodeEncodeError: 'charmap' codec can't encode characters in position 0-2: character maps to <undefined>
FAILED: zephyr/cmake/reports/CMakeFiles/rom_report
cmd.exe /C "cd /D C:\project\build && C:\ncs\toolchains\v2.0.2\opt\bin\python.exe C:/ncs/v2.0.2/zephyr/scripts/footprint/size_report -k C:/project/build/zephyr/zephyr.elf -z C:/ncs/v2.0.2/zephyr -o C:/project/build --workspace=C:/ncs/v2.0.2 -d 99 rom"
ninja: build stopped: subcommand failed.
FATAL ERROR: command exited with status 1: 'C:\ncs\toolchains\v2.0.2\opt\bin\cmake.EXE' --build 'C:\project\build' --target rom_report
```

--- a/scripts/footprint/size_report
+++ b/scripts/footprint/size_report
@@ -12,6 +12,7 @@ Process an ELF file to generate size report on RAM and ROM.
 """
 
 import argparse
+import locale
 import os
 import sys
 import re
@@ -229,7 +230,7 @@ def get_die_filename(die, lineprog):
         directory = lineprog.header['include_directory'][dir_index - 1]
         filename = os.path.join(directory, file_entry.name)
 
-    path = Path(filename.decode())
+    path = Path(filename.decode(locale.getpreferredencoding()))
 
     # Prepend output path to relative path
     if not path.is_absolute():

--- a/scripts/snippets.py
+++ b/scripts/snippets.py
@@ -311,7 +311,7 @@ def write_cmake_out(snippets: Snippets, cmake_out: Path) -> None:
     detail and are not meant to be used outside of snippets.cmake.'''
     if not cmake_out.parent.exists():
         cmake_out.parent.mkdir()
-    with open(cmake_out, 'w') as f:
+    with open(cmake_out, 'w', encoding="utf-8") as f:
         SnippetToCMakePrinter(snippets, f).print_cmake()
 
 def main():

